src/cephadm/cephadm

```diff
@@ -3334,6 +3334,7 @@ def extract_uid_gid_monitoring(daemon_type):
     if daemon_type == 'prometheus':
         uid, gid = extract_uid_gid(file_path='/etc/prometheus')
     elif daemon_type == 'node-exporter':
+        call_throws(['podman', 'pull', args.image])
```

Comment:
this will break a docker deployment. 

maybe we should be have the mgr do a `cephadm login` and `cephadm pull` before a `cephadm deploy ...` ?

Reply:
we doing the login already in check_host: 

https://github.com/ceph/ceph/blob/5a0090a38be02287514b8fc7902f3b09189484ea/src/pybind/mgr/cephadm/serve.py#L123-L126

Reply:
mmmm. but this happens in the bootstrap node ..... 
In the command_bootstrap it is included the "podman/docker login command" but only  when the bootstrap command provides credentials ... 

https://github.com/ceph/ceph/blob/50f580c6e3b8853e87a2046d3c80d0e51964929b/src/cephadm/cephadm#L2886

@Daniel-Pivonka : I saw in your pastebin that you haven't used credentials.... maybe this is the problem.

Can you check if it works using:

```./cephadm bootstrap --mon-ip ....... --registry-url registry.redhat.io  --registry-username xxxxxx --registry-password xxxxx```

Reply:
same thing happens if the flags are used during bootstrap   https://pastebin.com/Dt9v8KFK